aws-toolkit-vscode.code-workspace

```diff
@@ -1,19 +1,19 @@
 {
-	"folders": [
-		{
-			"path": "."
-		},
-		{
-			"path": "packages/toolkit"
-		},
-		{
-			"path": "packages/core"
-		},
-		{
-			"path": "packages/amazonq"
-		}
-	],
-	"settings": {
-		"typescript.tsdk": "node_modules/typescript/lib"
-	}
-}
\ No newline at end of file
+    "folders": [
```

Comment:
is this a formatting-only change? please revert the formatting, and only include the relevant changes.